### PR TITLE
Update fonts for hash icon

### PR DESCRIPTION
**Pull request by reedsa**
Add Hash icon to fonts and class vars.
Document the procedure to update icons in ui-c.

[PEEPS-1478](https://jira.sendgrid.net/browse/PEEPS-1478)

## Merge Checklist

- [X] Includes link to associated Github Issue or JIRA ticket
- [X] Has descriptive PR title and body
- [X] All changes are backwards compatible
- [ ] QA'd by the SendGrid Style Guide designer
- [ ] Code reviewed by a SendGrid frontend lead

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,16 @@ More information: https://semver.org/
...
 
+## Updating Icons
+
+To update icons with the latest from StyleGuide follow these steps to change the font files and update necessary classes and variables.
+
+1. Download `eot`, `ttf`, `woff`, `woff2` files from [StyleGuide Icons](https://github.com/sendgrid/style-guide/tree/master/packages/style-guide/fonts/icons)
+2. Replace files in the [icons](src/styles/fonts/icons) directory with those that were downloaded
+3. Add new variable to the [variables](src/styles/global/variables.scss) file with the unicode value found in the [StyleGuide SVG](https://raw.githubusercontent.com/sendgrid/style-guide/master/packages/style-guide/fonts/icons/style-guide-icons.svg) file
+4. Add new style class for the icon in [icon.module.scss](src/styles/icon.module.scss) prepended with `.sg-icon-` and use the variables that were created in step 3
+5. Run `npm run update-icon-types` to add the new icon as a type in [icons.ts](src/types/icons.ts)
```
Comment (Ditofry) [inline README.md]: you're a good person

File: package-lock.json [hunk 2]
```diff
@@ -1657,6 +1657,20 @@
...
+          "dev": true,
+          "requires": {
+            "browserslist": "^4.5.4",
+            "caniuse-lite": "^1.0.30000957",
+            "normalize-range": "^0.1.2",
+            "num2fraction": "^1.2.2",
+            "postcss": "^7.0.14",
+            "postcss-value-parser": "^3.3.1"
+          }
+        },
```
Comment (Ditofry) [inline package-lock.json]: Woah, how did all of this stuff change?

Comment (reedsa) [reply package-lock.json]: Oops, good catch. Was on the wrong version of node!

**Comment by thinkingserious** [comment]
Hello @reedsa,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**